tidy.js

```diff
@@ -0,0 +1,88 @@
+var tidy = require('tidy-html5').tidy_html5
```

Comment:
The tidy-html5 npm does not ship with a command line interface, so we need to wrap it in a script file :(